baseband/vlbi_base/base.py

```diff
@@ -79,6 +81,11 @@ def __init__(self, fh_raw, header0, unsliced_shape, bps, complex_data,
         self._subset = subset
         self._sample_shape = self._get_sample_shape(subset_wrapints)
 
+    @property
```

Comment:
Even though the private `_fill_value` attribute is set in the base, I think it makes sense to only expose it in the reader; for the writer at least currently it is unused so would only lead to confusion. (I'm wondering whether I can do the hierarchy better such that the writer doesn't even have the attribute, but that's definitely beyond the scope of this PR!)